contracts/src/interfaces/IDepositQueue.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
Good catch, wasn't sure about this since it came from renzo.  Should we keep it @jalextowle?

Reply:
In my experience you leave it the same from where you got it if you can.

Reply:
We don't want a BUSL license in the repo. To avoid this, I'd say we consolidate all of this into an `IRenzo` interface that includes all of the functions. That way, they didn't write the code -- we did.

Reply:
took that approach and moved everything to IRenzo.sol